recipes/findorb/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+CFLAGS="$CFLAGS -I $CONDA_PREFIX/include"
+CXXFLAGS="$CXXFLAGS -I $CONDA_PREFIX/include -L $CONDA_PREFIX/lib"
+LDFLAGS="$LDFLAGS -L $CONDA_PREFIX/lib"
```

Comment:
I suspect this isn't needed. Regardless it's definitely wrong as `CONDA_PREFIX` isn't well defined in this context.
```suggestion
```

Reply:
You're very much right, thanks! This was a left-over from when ncurses were accidentally left in the `requirements.build` section. Will fix (remove).